src/styles/utils/_extends.scss

```
@@ -1,4 +1,23 @@
-%h1 {
-  font-family: "Roboto", sans-serif;
-  font-weight: 400;
+%conteiner {
+  max-width: 1280px;
+  padding: 0 120px;
+  margin: 0 auto;
```

```suggestion
  margin: 0 auto;
```
You cannot use outer spacings for BEM blocks. This mixin is used in many blocks in your code (gallery, contacts, and about).
Try to center the content in some other way. For example, you can create a mix of classes. Create an element of `main` called `main__container` which will center your content